gulpfile.js

```diff
@@ -924,9 +925,44 @@ gulp.task('jsdoc', function (done) {
   });
 });
 
+gulp.task('lib', ['buildnumber'], function () {
+  function preprocess(content) {
+    content = preprocessor2.preprocessPDFJSCode(ctx, content);
+    var removeCjsSrc =
+      /^(var\s+\w+\s*=\s*require\('.*?)(?:\/src)(\/[^']*'\);)$/gm;
+    content = content.replace(removeCjsSrc, function (all, prefix, suffix) {
+      return prefix + suffix;
+    });
+    return licenseHeader + content;
+  }
+  var versionInfo = getVersionJSON();
+  var ctx = {
+    rootPath: __dirname,
+    saveComments: false,
+    defines: builder.merge(DEFINES, {
+      GENERIC: true,
+      BUNDLE_VERSION: versionInfo.version,
+      BUNDLE_BUILD: versionInfo.commit
+    })
+  };
+  var licenseHeader = fs.readFileSync('./src/license_header.js').toString();
+  var preprocessor2 = require('./external/builder/preprocessor2.js');
+
+  return merge([
+    gulp.src([
+      'src/{core,display,shared}/*.js',
```

Comment:
Nit: I don't feel strongly about it, however you may wish to skip [src/shared/cffStandardStrings.js](https://github.com/mozilla/pdf.js/blob/master/src/shared/cffStandardStrings.js) and [src/shared/fonts_utils.js](https://github.com/mozilla/pdf.js/blob/master/src/shared/fonts_utils.js) here, since neither of those are used in production currently (or anywhere else for that matter).